integration/test/releasehandling/release_handling_test.go

```diff
@@ -111,4 +113,40 @@ func TestReleaseHandling(t *testing.T) {
 			t.Fatalf("err == %v, want %v", err, nil)
 		}
 	}
+
+	// Verify that aws-operator.4.6.0 app CR exists.
+	{
+		o := func() error {
+			_, err := config.K8sClients.G8sClient().ApplicationV1alpha1().Apps("giantswarm").Get("aws-operator.4.6.0", metav1.GetOptions{})
+			if err != nil {
+				return microerror.Mask(err)
+			}
+
+			return nil
+		}
+		b := backoff.NewMaxRetries(30, 1*time.Second)
+
+		err := backoff.Retry(o, b)
+		if err != nil {
+			t.Fatalf("err == %v, want %v", err, nil)
+		}
+	}
```

Comment:
Can we make it the same way as in `multireleasehandling`? That is a bit more compact.
https://github.com/giantswarm/release-operator/pull/107/files#diff-d335cb88aca0d0ed82d92beb2ae7f0e8R119

Reply:
good shout. updated this